python/ray/serve/tests/test_deploy.py

```diff
@@ -396,7 +395,8 @@ def call(block=False):
                     "block": block
                 }).text
 
-        return ret.split("|")[0], ret.split("|")[1]
+        rets = {key: value for key, value in enumerate(ret.split("|"))}
```

Comment:
What's the reason we need to use a list comprehension and ``enumerate`` here instead of directly using ``split``? Is it a Windows-related change?

Reply:
N.B. - https://github.com/ray-project/ray/pull/21391#discussion_r778682352 

I have explained the reason for this change in the above comment. I used dict for a two line change. I was if-else if checks earlier.

Reply:
Got it, I added a question to that thread. I want to make sure the corner case doesn't affect the test's correctness.

Reply:
N.B. - https://github.com/ray-project/ray/pull/21391#discussion_r779424234